exaudfclient/base/exaudflib/exaudflib.h

```diff
@@ -21,7 +21,7 @@ bool send_run(zmq::socket_t &socket);
 void send_undefined_call(zmq::socket_t &socket, const std::string& fn);
 bool send_done(zmq::socket_t &socket);
 void send_finished(zmq::socket_t &socket);
-
+bool send_return(zmq::socket_t &socket, const char* result);
```

Comment:
was this missing before?

Reply:
Actually, all high level socket functions already were declared here. Or I would create a new header file just for send_return or add the function here.

Reply:
A separate header would have the benefit, it has the same name as cc-file. So, it would be easier to find it.

Reply:
Done